etc/natron.profile

```diff
@@ -12,7 +12,7 @@ noblacklist /opt/natron
 
 include /etc/firejail/disable-common.inc
 include /etc/firejail/disable-devel.inc
-include /etc/firejail/disable-interpreters.inc
+#include /etc/firejail/disable-interpreters.inc
```

Comment:
Hi @bn0785ac Do you know which language(s) natron needs access to? We can allow access to those while still blocking access to the others in disable-interpreters.

Reply:
python :)

Reply:
Thanks!

Reply:
Fixed (natron runs but i got crazy logs)


TESTING warning: noblacklist /usr/lib/python3* not matched by a proper blacklist command in disable*.inc